src/app/shared/change-password-dialog/password-check.service.ts

```diff
@@ -0,0 +1,33 @@
+/** Angular Imports */
```

Comment:
shouldn't this service be deleted? the function is already there in dialog.component